tunedev/Banka
# #165634314 adds api documentation

#### What does this PR do?
This pr adds API documentation to this app

#### Description of Task to be completed?
> - Implement feedback gotten during app's review by LFA
> - Add documentation router
> - Update the build script to copy all the non js files

#### How should this be manually tested?
> - Clone this repo and cd into its root folder
> - Run `npm install` and `npm run devstart`
> - visit this URL http://localhost:3000/api-docs
> - You should see the app's api doumentation

#### Any background context you want to provide?
NA

#### What are the relevant pivotal tracker stories?
[#165634314](https://www.pivotaltracker.com/story/show/165634314)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/33371467/56865853-94119000-69ca-11e9-81d2-3a929d46db08.png)

#### Questions:
NA
